src/sqlfluff/rules/L061.py

```diff
@@ -0,0 +1,66 @@
+"""Implementation of Rule L061."""
+
+from typing import Optional
+
+from sqlfluff.core.parser.segments.raw import CodeSegment
+from sqlfluff.core.rules.base import BaseRule, LintFix, LintResult, RuleContext
+from sqlfluff.core.rules.doc_decorators import document_fix_compatible
+from sqlfluff.core.rules.functional import sp
+
+
+@document_fix_compatible
+class Rule_L061(BaseRule):
+    """Use ``!=`` instead of ``<>`` for "not equal to" comparisons.
+
+    | **Anti-pattern**
+    | ``<>`` means ``not equal`` but doesn't sound like this when we say it out loud.
+
+    .. code-block:: sql
+
+        SELECT * FROM X WHERE 1 <> 2;
+
+    | **Best practice**
+    | Use ``!=`` instead because it's sounds more natural and is more common in other
+    | programming languages.
+
+    .. code-block:: sql
+
+        SELECT * FROM X WHERE 1 != 2;
+
+    """
+
+    def _eval(self, context: RuleContext) -> Optional[LintResult]:
+        """Use ``!=`` instead of ``<>`` for ``not equal to`` comparison."""
```

Comment:
```suggestion
        """Use ``!=`` instead of ``<>`` for "not equal to" comparison."""
```
Match the change above